src/main/scala/scala/tools/refactoring/implementations/OrganizeImports.scala

```diff
@@ -559,3 +565,79 @@ abstract class OrganizeImports extends MultiStageRefactoring with TreeFactory wi
     }.transform(tree)
   }
 }
+
+class OrganizeImportsSpacer(val oiInstance: OrganizeImports) {
+  import oiInstance.global._
+  import scala.tools.nsc.interactive.Global
+
+  private def topMostClassOrModule(stats: List[Tree]): Option[ImplDef] = stats.find {
+    case classDefOrModuleDef: ImplDef => true
+    case _ => false
+  }.asInstanceOf[Option[ImplDef]]
+
+  private def linesDiffInOrigFile(classOrModule: Option[ImplDef], importt: Option[Import]): Option[Int] =
+    for {
+      cm <- classOrModule
+      imp <- importt
+      sourceFile = cm.pos.source
+      cmLine = sourceFile.offsetToLine(cm.pos.start)
+      impLine = sourceFile.offsetToLine(imp.pos.end)
+    } yield cmLine - impLine
+
+  private def lastImportInPackageScope(stats: List[Tree]): Option[Import] = topMostClassOrModule(stats).flatMap { topMost =>
```

Comment:
intentional, this is like "get import when top most found" in other case code would look like:
```scala
val topMost = top...(stats)
if (topMost.nonExists /* or its analog*/) Nil
val lastImport = last...(stats, topMost)
...
```
so there is no flow in processing
So do you want it?